docs/design.md

```diff
@@ -44,42 +50,44 @@ Receivers typically listen on a network port and receive telemetry data. Usually
 
 ```yaml
 receivers:
-  opencensus:
-    endpoint: "0.0.0.0:55678"
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
 
 service:
   pipelines:
     traces:  # a pipeline of “traces” type
-      receivers: [opencensus]
+      receivers: [otlp]
       processors: [memory_limiter, batch]
       exporters: [jaeger]
     traces/2:  # another pipeline of “traces” type
-      receivers: [opencensus]
+      receivers: [otlp]
       processors: [batch]
       exporters: [opencensus]
 ```
 
-In the above example “opencensus” receiver will send the same data to pipeline “traces” and to pipeline “traces/2”. (Note: the configuration uses composite key names in the form of `type[/name]` as defined in [this document](https://docs.google.com/document/d/1NeheFG7DmcUYo_h2vLtNRlia9x5wOJMlV4QKEK05FhQ/edit#)).
+In the above example `otlp` receiver will send the same data to pipeline `traces` and to pipeline `traces/2`. (Note: the configuration uses composite key names in the form of `type[/name]` as defined in [this document](https://docs.google.com/document/d/1NeheFG7DmcUYo_h2vLtNRlia9x5wOJMlV4QKEK05FhQ/edit#)).
```

Comment:
Can we get `this document` included in this repo and not on a gdoc of unknown provenance?

Reply:
See PR here https://github.com/open-telemetry/opentelemetry-collector/pull/6369